### ISSUE TITLE
Error Message when I want to import Testcollection 

### ISSUE THREAD

**Issue by erwinmoretz** [rangav/thunder-client-support#1330]
**Question:**

Hi @rangav, 

me and some other colleagues try to import testcollections and get an error message: 

![image](https://github.com/rangav/thunder-client-support/assets/112586589/1c074db9-4d45-451a-8136-0b253efe30fd)

do you have an idea, how to fix this? 

**Are you using the free version/paid version/evaluating:**

paid

**Your Team Size Using TC:**
10-15

**Comment by rangav** [comment]
Hi @erwinmoretz 

what is the collection format - postman, open api or TC?

**Comment by erwinmoretz** [comment]
its a TC Collection. Im working with the collection and exported it to a colleague. He has this issue

**Comment by rangav** [comment]
Can you try to import and see if it works?

**Comment by erwinmoretz** [comment]
Yes it worked. We figured out were the error was: 

if we shared or tried to import collections directly from our Repository it failed. 
Example this files:
![image](https://github.com/rangav/thunder-client-support/assets/112586589/99ab343a-977f-44e8-8d3e-6aea03e3928d)


When we Export and import the exported TC File it worked. 
Example this File: 
![image](https://github.com/rangav/thunder-client-support/assets/112586589/3d4e7401-9094-4ad6-b030-0f985b712705)
